docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>standard squared-Euclidean distance</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>maybe "The Sinkhorn algorithm <strong>uses</strong>"</p>